Mayuresh Jagtap/array/array.js

```diff
@@ -0,0 +1,70 @@
+// Define a function maxOfTwoNumbers that takes two numbers as arguments and returns the largest of them. Use the if-then-else construct available in Javascript. You'll have to remember your pre-work, or do some googling to figure this out.
+const maxOfTwo = function (a,b){
+  if (a > b){
+    return a;
+  } else {
+    return b;
+  }
+};
+console.log (maxOfTwo (2,6));
+
+
+
+// Define a function maxOfThree that takes three numbers as arguments and returns the largest of them.
+const maxOfThree = function (a,b,c) {
+  if (a > b && a > c) {
+    return a;
+  } else if ( b > a && b > c) {
+    return b;
+  } else {
+    return c;
+  }
+};
+console.log (maxOfThree (23,54,9));
+
+
+
+
+
+// Write a function that takes a character (i.e. a string of length 1) and returns true if it is a vowel, false otherwise.
+const vowel = function (charVariable) {
+  if (charVariable === 'a'|| charVariable === 'e'|| charVariable === 'i'|| charVariable === 'o'|| charVariable === 'u') {
+    return 'true, it is a vowel';
+  } else {
+    return 'false, it is not a vowel';
+  }
+};
+console.log (vowel ('z'));
```

Comment:
great code, nice use of the or pipes ||. One small nitpick here is to write functions like this `vowel('z')` instead of this `vowel ('z')`. same goes for numbers[i] on line 51 instead of numbers [i]. Should still work either way most of the time, just best practice I believe. Otherwise variable names all look great on my end, didn't see any issues